### ISSUE TITLE
Link to correct upstream azurerm provider

### ISSUE THREAD

**Issue by clstokes** [pulumi/docs#1395]
We link to the incorrect upstream azure provider in our docs. The text and link should be to https://github.com/terraform-providers/terraform-provider-azurerm, not the _classic_ provider - note: `azurerm` vs `azure`.

![image](https://user-images.githubusercontent.com/293763/61475491-c387c700-a93f-11e9-8e97-e59ff2692375.png)

**Comment by justinvp** [comment]
The problem is we assume the Pulumi package and Terraform package have the same name, but for Azure that's not the case.

https://github.com/pulumi/pulumi-terraform/blob/f106438be1cfc83d362d591dc216570dfda9a879/pkg/tfgen/docs.go#L168-L171

And for Python:

https://github.com/pulumi/docs/blob/fda97a66dfb880b7c7494d3c89bc89b5c5d81a04/tools/pydocgen/pulumi-docs.json#L21-L23

**Comment by lukehoban** [comment]
Note that `g.info.Name` contains the Terraform Provider name.

```
Name           string                     // the TF provider name (e.g. terraform-provider-XXXX).
```

**Comment by justinvp** [comment]
Yep, I already started on a fix.

**Comment by stack72** [comment]
@justinvp  I believe the PR I merged yesterday #1407 - fixes this right?

**Comment by justinvp** [comment]
#1407 fixes it for the Python docs, but the node docs require a release of Azure (that contains https://github.com/pulumi/pulumi-azure/pull/309) and GCP (that contains https://github.com/pulumi/pulumi-gcp/pull/182), and then regenerating node docs for those packages.

**Comment by stack72** [comment]
ok, I will take care of this as soon as I get to the office - we have some python build issues

**Comment by stack72** [comment]
@justinvp GCP - https://github.com/pulumi/docs/pull/1425

**Comment by stack72** [comment]
@justinvp Azure - https://github.com/pulumi/docs/pull/1427

**Comment by stack72** [comment]
Closed via #1425 and #1427
